packages/tasks/src/hardware.ts

```diff
@@ -31,7 +31,7 @@ export interface HardwareSpec {
 	memory?: number[];
 }
 
-export const DEFAULT_MEMORY_OPTIONS = [8, 16, 24, 32, 40, 48, 64, 80, 96, 128];
+export const DEFAULT_MEMORY_OPTIONS = [6, 8, 16, 24, 32, 40, 48, 64, 80, 96, 128];
```

Comment:
Should we change this line? It's not related IMO

Reply:
I agree, that should not be necessary.

Reply:
yup, sorry somehow thought that was needed at first but you're totally right. reverted it